src/Drivers/nlpMDS_ex4_raja_driver.cpp

```diff
@@ -55,7 +64,7 @@ static bool parse_arguments(int argc, char **argv,
     return false; //5 or more arguments
   }
 
-  if(self_check && n_sp!=400 && n_de!=100)
+  if(self_check && (n_sp!=400 || n_de!=100 || empty_sp_row) )
```

Comment:
Same here -- we could allow an empty row for tests and add third test fro example 4.